schema/User.ts

```diff
@@ -6,57 +6,40 @@ import {
   select,
   text,
 } from "@keystone-6/core/fields";
+import { permissions, rules } from '../access';
 
 export const User = list({
-  // access:
-  // ui:
+  access: {
+    create: () => true,
+    read: rules.canManageUsers,
+    update: rules.canManageUsers,
+    // only people with the permission can delete themselves!
```

Comment:
This is negotiable, I think our clients should also be able to delete the account but we should store their data nevertheless.